src/og/layer/WMS.js

```diff
@@ -36,16 +37,23 @@ import { XYZ } from "./XYZ.js";
  *     zIndex: 50,
  *     attribution: 'USA states - geoserver WMS example',
  *     version: "1.1.1",
- *     visibility: false }
+ *     visibility: false }, {
+ *     transparent: true,
+ *     sld: "style.sld"}	   
  * );
  *
  * @fires og.layer.XYZ#load
  * @fires og.layer.XYZ#loadend
  */
 class WMS extends XYZ {
-    constructor(name, options) {
+
+
+    constructor(name, options,extra ) {
         super(name, options);
 
+        this.extra = new URLSearchParams(extra).toString();

```

Comment:
Little code convenient, In this case, obvious that extra is a private field because the extra (input) is an object, but the field is a string. Please make it `_extra`. 

P.S.
No need to add get/set extra for now.

Reply:
Ok, I've add the space and change the field name.